src/saveload/saveload.h

```diff
@@ -399,6 +399,27 @@ struct ChunkHandler {
 /** A table of ChunkHandler entries. */
 using ChunkHandlerTable = span<const ChunkHandler>;
 
+/** A table of SaveLoad entries. */
+using SaveLoadTable = span<const struct SaveLoad>;
+
+class SaveLoadHandler {
```

Comment:
I guess the draftiness of this PR means that the comments here are still pending / that you're waiting on glx's work, right?

Reply:
it means a lot of things, but most that this is not the final form yet, and that I didn't do a pass myself on any coding guideliness :) I mostly created the PR so I could collect opinions on the way of doing stuff.